### PR TITLE
Update glossary.md

### PR DESCRIPTION
**Pull request by svoskobo-vmware-com**
Making minor changes, so that content follows VMware documentation standards.

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by dvonthenen** [comment]
The CI markdown flagged the following...
```
docs/book/glossary.md: 5: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/book/glossary.md: 6: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs/book/glossary.md: 15: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

File: docs/book/glossary.md [hunk 1]
```diff
@@ -2,42 +2,39 @@
 
 ## VM
 
-A VM is an abstraction of an operating system from the physical machine by creating a "virtual" representation of the physical hardware the OS expects to interact with,
-this includes but is not limited to CPU instruction sets, memory, BIOS, PCI buses, etc. A VM is an entirely self-contained entity and shares no components with the host OS.
-In the case of vSphere the host OS is ESXi (see below).
+A virtual machine (VM) is a software computer that, like a physical computer, runs an operating system and applications. The VM abstracts the operating system from physical hardware resources, with which the OS typically interacts, and creates virtual representations of these resources.
```
Comment (frapposelli) [inline docs/book/glossary.md:5]:
```suggestion
A virtual machine (VM) is a software computer that, like a physical computer, runs an operating system and applications. The VM abstracts the operating system from physical hardware resources, with which the OS typically interacts, and creates virtual representations of these resources.
```

File: docs/book/glossary.md [hunk 2]
```diff
@@ -2,42 +2,39 @@
 
 ## VM
 
-A VM is an abstraction of an operating system from the physical machine by creating a "virtual" representation of the physical hardware the OS expects to interact with,
-this includes but is not limited to CPU instruction sets, memory, BIOS, PCI buses, etc. A VM is an entirely self-contained entity and shares no components with the host OS.
-In the case of vSphere the host OS is ESXi (see below).
+A virtual machine (VM) is a software computer that, like a physical computer, runs an operating system and applications. The VM abstracts the operating system from physical hardware resources, with which the OS typically interacts, and creates virtual representations of these resources. 
+The virtual hardware resources include CPU, memory, disks, and so on. The VM is a self-contained entity and shares no components with the host OS.
```
Comment (frapposelli) [inline docs/book/glossary.md:6]:
```suggestion
The virtual hardware resources include CPU, memory, disks, and so on. The VM is a self-contained entity and shares no components with the host OS.
```

File: docs/book/glossary.md [hunk 3]
```diff
@@ -2,42 +2,39 @@
...
 
-vSphere is the product name of the two core components of the VMware Software Defined Datacenter (SDDC) stack, they are vCenter and ESXi. Each is discussed below in detail.
+vSphere is a VMware suite of virtualization products and technologies, among which ESXi and vCenter Server are the core components.
 
 ## ESXi
 
-ESXi is the hypervisor, or "host" OS that is used to run VMs on. ESXi provides strong separation between VMs and itself, providing strong security boundaries between the
-guest and host operating systems. ESXi can be used as a standalone entity, without vCenter but this is extremely uncommon and feature limited as without a higher level manager (vCenter)
-ESXi cannot provide its most valuable features, like High Availability, vMotion, workload balancing and vSAN (a software defined storage stack).
+ESXi is a VMware hypervisor, or host operating system, that installs on a physical server and enables VMs with different operating systems to run side-by-side. ESXi provides strong separation between VMs and itself, offering security boundaries between the guest operating systems and the host.
```
Comment (frapposelli) [inline docs/book/glossary.md:15]:
```suggestion
ESXi is a VMware hypervisor, or host operating system, that installs on a physical server and enables VMs with different operating systems to run side-by-side. ESXi provides strong separation between VMs and itself, offering security boundaries between the guest operating systems and the host.
```

**Comment by dvonthenen** [comment]
I didn't know they had a suggestion feature. awesome! super helpful @frapposelli 

thanks for the contribution @svoskobo-vmware-com

**Pull request merged**